AMDirT/validate/application/__init__.py

```diff
@@ -57,3 +63,72 @@ def check_multi_values(
             return False
         else:
             return True
+
+    def check_sample_accession(self, remote: Union[AnyStr, None] = None) -> bool:
+        """Check that sample accession are valid
+
+        Args:
+            remote (AnyStr | None, optional): Remote to check against. Defaults to None.
+        """
+        if not remote:
+            with open(get_json_path()) as f:
+                tables = json.load(f)
+            samples = tables["samples"]
+            for table in samples:
+                if self.dataset_name == Path(samples[table]).name:
+                    remote = samples[table]
+            if remote is None:
+                raise SystemExit(
+                    f"No remote found for {self.dataset} dataset, please provide one"
+                )
+        remote_samples = DatasetValidator(schema=self.schema_path, dataset=remote)
+        df_change = pd.concat([remote_samples.dataset, self.dataset]).drop_duplicates(keep=False)
+        df_change.drop_duplicates(inplace=True, keep='last', subset=list(df_change.columns)[:-1])
+        print(df_change)
```

Comment:
```suggestion
```

Reply:
Hey @jfy133 ,
Getting back to it now:
Both work for me, can you test again if this works for you ?

```bash
$ AMDirT validate -s ancientmetagenome-hostassociated_samples.tsv ancientmetagenome-hostassociated_samples_schema.json
ancientmetagenome-hostassociated_samples.tsv is valid
2023-02-24 16:08:34.800 INFO    AMDirT: ancientmetagenome-hostassociated_samples.tsv is valid
$ AMDirT validate -s -a ancientmetagenome-hostassociated_samples.tsv ancientmetagenome-hostassociated_samples_schema.json
ancientmetagenome-hostassociated_samples.tsv is valid
2023-02-24 16:08:41.262 INFO    AMDirT: ancientmetagenome-hostassociated_samples.tsv is valid
```